package.json

```diff
@@ -17,11 +17,11 @@
   "license": "BSD-3-Clause",
   "dependencies": {
     "depcheck": "^0.6.3",
-    "fs-extra": "^0.30.0",
-    "gulp-eslint": "^2.0.0",
-    "gulp-mocha": "^2.2.0",
-    "gulp-tslint": "^5.0.0",
-    "gulp-typescript": "^2.13.4",
+    "fs-extra": "^2.0.0",
+    "gulp-eslint": "^3.0.1",
```

Comment:
It'd be better to not rely on a patch version on a lower bound. If `^3.0.0` works, prefer it. The lower bound should be as low as possible.